utils/bigclown/bigclown-gateway/Makefile

```diff
@@ -6,24 +6,25 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=bigclown-gateway
 PKG_VERSION:=1.16.2
-PKG_RELEASE:=1
+PKG_RELEASE:=2
+
+PYPI_NAME:=bcg
+PKG_HASH:=ce7f27f372551c0beb3f8929af2d779417d9dcd0feaa2fa2dc49e87b1416c536
 
-PKG_SOURCE_URL:=https://codeload.github.com/bigclownlabs/bch-gateway/tar.gz/v$(PKG_VERSION)?
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_HASH:=9d3208bf4cffec507d992485104fcbba2b9bc02cf7b290dfe13f98e5916ee1ca
-PKG_LICENSE:=MIT
 PKG_MAINTAINER:=Karel Kočí <cynerd@email.cz>
-PKG_BUILD_DIR:=$(BUILD_DIR)/bch-gateway-$(PKG_VERSION)
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
 
+include ../../../lang/python/pypi.mk
 include $(INCLUDE_DIR)/package.mk
 include ../../../lang/python/python3-package.mk
 
 define Package/$(PKG_NAME)
   SECTION:=utils
   CATEGORY:=Utilities
   SUBMENU:=BigClown
-  URL:=https://github.com/bigclownlabs/bch-gateway
   TITLE:=BigCLown gateway
+  URL:=https://github.com/bigclownlabs/bch-gateway
   DEPENDS:= \
```

Comment:
Can you also add python3-click as a dependency here, since it is [listed](https://github.com/bigclownlabs/bch-gateway/blob/v1.16.2/requirements.txt#L1) in requirements.txt (which is read by setup.py)?

Reply:
Ok